Apromore-Commons/pom.xml

```diff
@@ -16,18 +16,17 @@
   <packaging>jar</packaging>
   <name>Apromore-Commons</name>
 
+  <properties>
```

Comment:
This is good, There are many new features. If you get the time and bandwitdth please do this for the core project. Not sure how much backward compatible it will be though.